### ISSUE TITLE
Spankbang is not work

### ISSUE THREAD

**Issue by Angeloblu83** [dobbelina/repository.dobbelina#1104]
Spankbang is not work fixed please

**Comment by Nash10se** [comment]
I agree that it's not working. Several times in the last few weeks I've gotten an error saying I should check the logs. (I have no idea where to find the logs) Then, the next day it usually works again. But starting about three days ago, it has stopped working and has not fixed itself.

**Comment by frankee123** [comment]
This site is one of my favorites as it has a lot of lesbian ass licking scenes which are my fav. I hope it can get fixed please. Ty

**Comment by imurreflection** [comment]
The site is accessible via VPN only. Try Netherlands or Singapore

**Comment by ErosVece** [comment]
Try this: https://github.com/dobbelina/repository.dobbelina/wiki/Cloudflare-Challenges-and-Flaresolverr-Integration-with-Cumination

**Comment by dvgm506** [comment]
Flaresolverr fixes it. Truecharts has it as one of their curated apps, so it was quick & easy to install on my truenas scale server.

**Comment by frankee123** [comment]
I entered spankbang.com under the advanced settings for address where flaresolver is running, and I get an error trying to open spankbang. When I click on it it says cloudflare detected, retrying with flavesolverr..and I get an error. Help please?

**Comment by ErosVece** [comment]
> I entered spankbang.com under the advanced settings for address where flaresolver is running, and I get an error trying to open spankbang. When I click on it it says cloudflare detected, retrying with flavesolverr..and I get an error. Help please?

You need to enter there the host where you have Flaresolverr installed.
For example `http://localhost:8191/v1' when you have it installed on the device where Kodi is running

**Comment by frankee123** [comment]
Thank you. I'm sorry, I just don't understand where do I find the link you are telling me about? Do I have to download flaresolver from somewhere do you mean?

**Comment by ErosVece** [comment]
Here :)

![image](https://github.com/dobbelina/repository.dobbelina/assets/37535332/dc6f116d-abd5-42f2-971e-6549e3c93c7a)

**Comment by frankee123** [comment]
Thx, I'll try and figure out where exactly to download a link on my android. I don't see one clearly for that when I click the link.

**Comment by ErosVece** [comment]
There is no Flaresolverr download for it to run on Android.
There are ways to run Docker on Android (root and non-root), and you can run the Flaresolverr container in that. But it ain't easy.
Search for: docker on android

But you can use a pc or a home server or a NAS that can run applications, and point Cumination to that.

**Comment by frankee123** [comment]
Ah ok thank you so much my friend. I just consider myself sool then lmao.

**Comment by dvgm506** [comment]
Could even be done with a raspberry pi.

**Comment by Johnpoke15** [comment]
This site is amazing and it's a real bummer it still doesn't work.

**Comment by jetsetcarwash** [comment]
Hi, no idea if this is right place but I too cannot get spankbang to work on cumination on Kodi 20? Any ideas??
